### ISSUE TITLE
False positive

### ISSUE THREAD

**Issue by ghost** [disposable/disposable#86]
Hello!

simplelogin.com
simplelogin.fr
aleeas.com
smail.me 
pm.me

I can see that you’d service think that it is a temporary mail, but it is NOT.

The SimpleLogin service (https://simplelogin.io/) is designed to hide a user's real email address, and protect it from spam. All addresses created by SimpleLogin are perfectly normal and have the ability to both receive and send emails.

Also, all of them are owned by the most famous encrypted email provider Proton AG (https://proton.me, ex [https://protonmail.com)]https://protonmail.com). You can check it just by Whois.

Please unblock the mail domains I indicated above

**Comment by jasonkarns** [comment]
Unfortunately, it seems others are trying to go even further and block _more_ simplelogins :/ #95 😞

**Comment by jasonkarns** [comment]
simplelogin is also noted on #81

**Comment by smeinecke** [comment]
Duplicate of #81
